src/test/java/model/CarTest.java

```diff
@@ -0,0 +1,25 @@
+package model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CarTest {
+
+    @Test
+    void shouldMove() {
+        Car testCar = Car.carWithName(new CarName("test"));
+        CarPosition originPosition = testCar.getCarPosition();
+
+        testCar.move();
+
+        assertThat(testCar.getCarPosition().compareTo(originPosition)).isPositive();
+    }
+
+    @Test
+    void shouldNotMove() {
+        Car testCar = Car.carWithName(new CarName("test"));
+
+        assertThat(testCar.getCarPosition()).isEqualTo(testCar.getCarPosition());
```

Comment:
이 조건은 `getCarPosition()` 이 임의의 값을 반환하지 않는다면 항상 참인 조건이 될 것 같아요.
아래와 같이 변경해보면 어떨까요?
```suggestion
        assertThat(testCar.getCarPosition()).isEqualTo(CarPosition.ZERO);
```